src/script/dashboard_e2e_tests.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+#
+# dashboard_e2e_tests.sh
+#
+# Run the Ceph Dashboard E2E tests against a real Ceph cluster
+#
+# CAVEAT: do *not* run this script as root, but *do* run it as a user with
+# passwordless sudo privilege.
+#
+# TODO: in its current form, this script assumes the Ceph cluster is
+# "pristine". More work is needed to achieve idempotence.
+
+if [ "$EUID" = "0" ] ; then
+    echo "$0: detected attempt to run script as root - bailing out!"
+    exit 1
+fi
+
+if [[ ! $(arch) =~ (i386|x86_64|amd64) ]]; then
+    echo "$0: this script uses Google Chrome, which is only available on amd64 - bailing out!"
```

Comment:
Not sure if this message matches the if-condition? Maybe use "x86 architectures" instead?